app/public/js/login.js

```diff
@@ -1,15 +1,11 @@
 $(document).ready( function () {
-    if (!firebase.apps.length) {
-        firebase.initializeApp(config);
-    }
-
```

Comment:
This bit isn't actually being used anymore since it's still in the frontend.  Jason moved the login for firebase to routes/admin-login.js (now routes/login.js), so it's obselete.  It probably should have been deleted back then so I figured I'd just do it now.  If it causes a merge error it should be an easy fix, right?  Since it isn't needed?